src/client/pydaos/raw/daos_cref.py

```diff
@@ -275,3 +299,9 @@ class Logfac:
     INFO = 1
     WARNING = 2
     ERROR = 3
+
+class CheckSumFlag:
```

Comment:
(pylint-missing-docstring) Missing class docstring
(pylint-too-few-public-methods) Too few public methods (0/2)

Reply:
Known problem with this file. We create class for storage purpose only... Known errors which can be discarded.